site/zenodo_rdm/redirector.py

```diff
@@ -165,6 +174,25 @@ def record_export_view():
     return target
 
 
+def legacy_record_export_view():
+    """Implements redirector view function for legacy values of record export.
+
+    The following routes are redirected as follows:
+        -  /records/<pid_value>/export/<legacy_export_format> -> GET /record/<pid_value>/export/<export_format>
+
+    :return: url for the view 'invenio_app_rdm_records.record_export'
+    :rtype: str
+    """
+
+    args = request.view_args
+    export_format = request.path.split("/")[-1]
```

Comment:
is the format also available in the view arguments? i.e. `args["export_format"]`

Reply:
No, as I had to add the full path in the config like this `/records/<pid_value>/export/hx`, otherwise I got into a redirect loop!

Reply:
Shouldn't the format be available in `request.args` or `request.view_args`?  Like [here](https://github.com/zenodo/zenodo-rdm/blob/08d21c983ec5218bc42a7e990db01369798029b9/site/zenodo_rdm/redirector.py#L102)